### PR TITLE
Set up BlockNumberUpdater for Arbitrum

### PR DESCRIPTION
**Pull request by mateuszradomski**
Resolves L2B-1646

### CODE REVIEW

**Review by sdlyy — APPROVED** [review]
Looks sweet, looking forward to merging this boy, so I can prevent some of the duplicates within work dependencies

File: packages/backend/src/modules/tvl/ArbitrumTvl.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  // #endregion
+
+  // #region updaters
+
+  const arbiscanBlockNumberUpdater = new BlockNumberUpdater(
+    arbiscanClient,
+    db.blockNumberRepository,
+    clock,
+    logger,
+    ChainId.ETHEREUM,
```
Comment (sdlyy) [inline packages/backend/src/modules/tvl/ArbitrumTvl.ts:39]: Double check if it shouldn't be `ChainId.ARBITRUM`

Comment (mateuszradomski) [reply packages/backend/src/modules/tvl/ArbitrumTvl.ts]: Good catch!

File: packages/backend/src/modules/tvl/TvlModule.ts [hunk 2]
```diff
@@ -40,113 +28,80 @@ export function createTvlModule(
...
 
   // #endregion
 
-  const start = async () => {
-    if (!config.syncEnabled) {
-      logger.info('TVL sync disabled')
-      return
-    }
+  const modules: (ApplicationModule | undefined)[] = [
+    createEthereumTvlModule(db, priceUpdater, config, logger, http, clock),
```
Comment (mateuszradomski) [inline packages/backend/src/modules/tvl/TvlModule.ts:91]: @antooni  Shouldn't we be creating the Arbitrum TVL module here as well? Why skip it?

File: packages/backend/src/modules/tvl/TvlModule.ts [hunk 3]
```diff
@@ -40,113 +29,81 @@ export function createTvlModule(
...
-    aggregatedReportRepository,
-    reportStatusRepository,
-    clock,
-    config.projects,
-    logger,
-  )
 
   // #endregion
+
   // #region api
```
Comment (michalsidzej) [inline packages/backend/src/modules/tvl/TvlModule.ts:71]: What's the deal with those extra spaces? 
```suggestion
  // #endregion
  // #region api
```

File: packages/backend/src/modules/tvl/TvlModule.ts [hunk 4]
```diff
@@ -40,113 +29,81 @@ export function createTvlModule(
...
 
   // #endregion
 
-  const start = async () => {
-    if (!config.syncEnabled) {
-      logger.info('TVL sync disabled')
-      return
-    }
+  const modules: (ApplicationModule | undefined)[] = [
+    createEthereumTvlModule(db, priceUpdater, config, logger, http, clock),
```
Comment (michalsidzej) [inline packages/backend/src/modules/tvl/TvlModule.ts:92]: WYT about renaming this to `Submodule`? It's one step deeper than module, so would be less confusing.
```suggestion
    createEthereumTvlSubmodule(db, priceUpdater, config, logger, http, clock),
```

**Review by michalsidzej — COMMENTED** [review]
Overall looks good, gj

**Review by antooni — APPROVED** [review]

File: packages/backend/src/modules/tvl/TvlModule.ts (hunk 4 above)
Comment (mateuszradomski) [reply packages/backend/src/modules/tvl/TvlModule.ts]: I like that idea! I've renamed these functions

**Pull request merged**